### PR TITLE
Memcached rewrite

### PR DESCRIPTION
**Pull request by bradleycamacho**

### CODE REVIEW

File: src/install/config/memcached.yaml
Comment (akristen) [inline src/install/config/memcached.yaml:1]: dont forget to update mongodb --> meme cashed

Comment (akristen) [reply src/install/config/memcached.yaml:1]: actually i dont know if you have to but im calling it out anyway

File: src/install/memcached/intro.mdx
Comment (akristen) [inline src/install/memcached/intro.mdx:8]:
```suggestion
Our Memcached [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your Memcached instance. Once instrumented, you can use New Relic to aggregate and visualize key performance metrics at the instance and slab levels.
```

Comment (akristen) [reply src/install/memcached/intro.mdx:8]: Once integrated or once instrumented, whatever is more precise. I guess we're integrating a tech into the agent, which instruments... your call

File: src/install/memcached/intro.mdx
Comment (akristen) [inline src/install/memcached/intro.mdx:11]: for some reason, this content isn't appearing in the deploy preview from netlify. am going to open it up in the local host but wanted to call that out

File: src/install/memcached/default-requirements.mdx
Comment (akristen) [inline src/install/memcached/default-requirements.mdx:6]: i think style guide has us do "or higher", which is niche knowledge i know from EOL comms .

also wondering if we should beef it up by including the infra agent version they need? might be redundant so am just riffing

File: src/install/memcached/linux/install-apt.mdx
Comment (akristen) [inline src/install/memcached/linux/install-apt.mdx:14]: indent the last set of backticks here. it's adding a secondary / random (i think) code block on the preview that reads:

`export const _frontmatter = {"componentType":"default","headingText":"Download using APT"}`

File: src/install/memcached/linux/install-apt.mdx
Comment (akristen) [inline src/install/memcached/linux/install-apt.mdx:6]:
```suggestion
From the command line, run these commands to update APT and install the integration:
```

File: src/install/memcached/intro.mdx
Comment (akristen) [reply src/install/memcached/intro.mdx:10]: fixed!

File: src/install/memcached/default-requirements.mdx
Comment (akristen) [reply src/install/memcached/default-requirements.mdx:6]: poking around their repo, they also say there's no limitations on MacOS versions. you could add that, too.

they have different instructions on their repo for GoLang, tho. wanted to just call that out in case we need to update some instructions: https://github.com/newrelic/nri-memcached?tab=readme-ov-file#building

File: src/install/memcached/linux/install-linux.mdx
Comment (akristen) [inline src/install/memcached/linux/install-linux.mdx:6]: markdown is rendering all steps in 'configure integration' as 1s

File: src/install/config/memcached.yaml
Comment (akristen) [inline src/install/config/memcached.yaml:47]: i believe live 47 is telling the branching doc to do a default path for Apt, which is messing with the doc in its neutral, reset state. Gives it all sorts of buggy behaviors

File: src/install/config/memcached.yaml
Comment (akristen) [inline src/install/config/memcached.yaml:1]: I don't see `src/install/memcached/default-configuration.mdx` in this yaml but it exists in the folder.

File: src/install/config/memcached.yaml
Comment (akristen) [reply src/install/config/memcached.yaml:47]: I tried removing this in the local to see if it worked but it didn't. idk maybe you want it to do that, but am calling this out anyway

File: src/install/memcached/windows/install-windows.mdx
Comment (akristen) [inline src/install/memcached/windows/install-windows.mdx:12]: these are displaying as two step 1s. this should be fixed if you just make an ordered list with 

1.
1.
1.

File: src/install/memcached/windows/install-windows.mdx
Comment (akristen) [inline src/install/memcached/windows/install-windows.mdx:8]: indent so theyre children of the step (?? unless it's the opposite from our indent sweep)

Comment (akristen) [reply src/install/memcached/windows/install-windows.mdx:8]: actually u should explain this to me if i'm wrong lol i know it was a team meeting topic but i am doubting i processed that section correctly

File: src/install/memcached/find-data.mdx
Comment (akristen) [inline src/install/memcached/find-data.mdx:6]: This def needs more context. i know what this means, but let's say you're a new user to New Relic. Where do i go to see this relationship? You might want to point them to the data explorer tool via pathing, and give a high level breakdown of how to actually find their data (data explorer) and use it (query your data).

afaik all of these actions are housed in the Query your data component nowadays

File: src/install/memcached/linux/install-yum.mdx
Comment (akristen) [inline src/install/memcached/linux/install-yum.mdx:6]:
```suggestion
From the command line, run these commands:
```

Comment (akristen) [reply src/install/memcached/linux/install-yum.mdx:6]: i think i made suggestions on another one of these parallel components to unspool the directions a bit more. if you go w/ em, don't forget to add those suggestions to these other ones

File: src/install/memcached/linux/install-zypper.mdx
Comment (akristen) [inline src/install/memcached/linux/install-zypper.mdx:6]:
```suggestion
From the command line, run these commands:
```

File: src/install/memcached/linux/install-tarball.mdx
Comment (akristen) [inline src/install/memcached/linux/install-tarball.mdx:20]: looks a little janky in the preview

**Review by akristen — CHANGES_REQUESTED** [review]
ok i left some comments on possibly weird behaviors among other things!! lmk when you need a second review/approval

File: src/install/config/memcached.yaml
Comment (bradleycamacho) [reply src/install/config/memcached.yaml:47]: I did intend it as we want all steps to present some sort of "default", and APT is a nice default state for it to be.

**Pull request merged**
